test/SignerBondsEscrow.test.js

```diff
@@ -0,0 +1,95 @@
+const chai = require("chai")
+const expect = chai.expect
+
+const { ZERO_ADDRESS } = require("./helpers/contract-test-helpers")
+
+describe("SignerBondsEscrow", () => {
+  let owner
+  let depositor
```

Comment:
I'd call them `governance` and `riskManager` to clearly indicate how relationships between components work - who is depositing and who is eligible to withdraw.

Reply:
See https://github.com/keep-network/coverage-pools/pull/39/commits/d8627cd094194912d5ca1a8b9dab2aa36bc6fd61